.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.14.1
```

Comment:
Where do we need it? You can set a required node version in the `package.json`. 

Why not a 17.x? LTS? If so, we are not using node as a server, so we can use the newest version.

Reply:
LTS is a moving target, so may break in future. 

I've added engines list in the package.json

Reply:
`.tool-versions` is used by `asdf` - a convenient tool to manage toolchain versions. we use it in many other projects, it works similar to `nodenv` or `nvm` but handle other toolchains

Reply:
The `engines` are added in the package nested config, but this file is in the root...

Reply:
I checked `asdf`, now I understand. However, I don't get this part:

> LTS is a moving target, so may break in future.

It seems to answer why we don't use LTS, but 16.x is LTS indeed :P